scripts/test/hyriseBenchmarkTPCH_test.py

```diff
@@ -1,34 +1,38 @@
 #!/usr/bin/env python3
 
 from hyriseBenchmarkCore import *
+from compareBenchmarkScriptTest import *
 
 # This test runs the binary hyriseBenchmarkTPCH with two different sets of arguments.
 # During the first run, the shell output is validated using pexpect.
 # After the first run, this test checks if an output file was created and if it matches the arguments.
 # During the second run, the shell output is validated using pexpect
 # and the test checks if all queries were successfully verified with sqlite.
 def main():
+  build_dir = initialize()
+  compare_benchmarks_path = f'{build_dir}/../scripts/compare_benchmarks.py'
+  output_filename = f"{build_dir}/tpch_output.json"
 
-  return_error = False
+  return_error = False  
 
   arguments = {}
   arguments["--scale"] = ".01"
   arguments["--use_prepared_statements"] = "true"
   arguments["--queries"] = "'1,13,19'"
   arguments["--time"] = "10"
   arguments["--runs"] = "-1"
-  arguments["--output"] = "'json_output.txt'"
   arguments["--mode"] = "'Shuffled'"
   arguments["--encoding"] = "'Dictionary'"
   arguments["--compression"] = "'Fixed-size byte-aligned'"
   arguments["--indexes"] = "true"
   arguments["--scheduler"] = "false"
   arguments["--clients"] = "1"
   arguments["--dont_cache_binary_tables"] = "true"
+  arguments["--output"] = output_filename
 
-  benchmark = initialize(arguments, "hyriseBenchmarkTPCH", True)
+  benchmark = run_benchmark(build_dir, arguments, "hyriseBenchmarkTPCH", True)
 
-  benchmark.expect_exact("Writing benchmark results to 'json_output.txt'")
+  benchmark.expect_exact(f"Writing benchmark results to '{output_filename}'")
```

Comment:
What does the `f` do?

Reply:
Python f-strings. The quasi successor to `"{}".format('a')` strings (https://realpython.com/python-f-strings/)